cmd/print.go

```diff
@@ -284,7 +284,7 @@ func (pr *PrintRunner) getProviderSpecificFlags() map[string]map[string]string {
 	for flagName, value := range pr.providerSpecificFlags {
 		provider, found := lo.Find(pr.providers, func(p string) bool { return strings.HasPrefix(flagName, fmt.Sprintf("%s-", p)) })
 		if !found {
-			log.Printf("Warning: Ignoring flag %s as it does not match any of the providers", flagName)
+			// log.Printf("Warning: Ignoring flag %s as it does not match any of the providers", flagName)
```

Comment:
thats noisy as there are some default flags that are configured. I'd like to have something like that for flags that the user explicitly provided.

@guicassolato what do you think?

Reply:
That makes total sense, @LiorLieberman.

The warning was added in the best intention of letting the user know a provider-specific flag was supplied out of context, but if it has the potential to create more confusion than actually help, then we should probably skip it.

Reply:
can we remove this line altogether, instead of commenting it?